spec/rails_helper.rb

```diff
@@ -80,7 +80,7 @@
   # config.include SupportHelper,    type: :feature
   # config.include SupportHelper,    type: :request
   config.include FactoryBot::Syntax::Methods
-  # config.include WaitForAjax, type: :feature
+  config.include WaitForAjax, type: :feature
```

Comment:
See comment above

Reply:
Thanks, i've removed the wait_for_ajax completely and it passes